ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -98,9 +105,11 @@ def __init__(self, instance):
             "connection_name:{}".format(self.connection_name),
         ]  # type: List[str]
         tags.extend(custom_tags)
-        if host or port:
+        if host and not override_hostname:
             # 'host' is reserved and 'mq_host' is used instead
-            tags.extend({"mq_host:{}".format(host), "port:{}".format(port)})
+            tags.append("mq_host:{}".format(host))
+        if port:
```

Comment:
With this change, if port is not specified, we won't submit the tag `port:none`. Do you think we should continue sending the tag?

Reply:
oh, you're right

Reply:
Ah I forgot to note this before, but the same is still true for `mq_host` and `port:None`